tasks/48_follow-the-spy/follow-the-spy.js

```diff
@@ -0,0 +1,15 @@
+export const followTheSpy = (arr) => {
+  const route = [...arr[0]];
+  const el = arr.slice(1);
+  while (el.length) {
+    let lastRoute = route.slice(-1)[0];
+    for (let i = 0; i < el.length; i++) {
```

Comment:
it is also possible that it would be better to use find method here to find a route where last elements in route will equal first one in arr